src/App.css

```diff
@@ -1,3 +1,14 @@
 .app {
-  color: aquamarine;
+  background-color: #fafafa;
+}
+
+.btn {
+  padding: 1em 3em;
+  cursor: pointer;
```

Comment:
- [optional] please add some extra margin/space to the bottom to the form can be well visible as in the original design